README.md

```diff
@@ -29,6 +29,14 @@ joining our [public Slack channel.](https://adhocteam-public.herokuapp.com)
 We think a sign of a good developer is one who asks questions sooner, rather
 than later. We are happy to help guide you to the right solution.
 
+On Overengineering
```

Comment:
Late to the party - isn't this already covered in [A quick tip](https://github.com/adhocteam/homework#a-quick-tip) section?

> we tend to favor solutions that clearly and concisely solve the problem at hand. We're not looking for an over-engineered solution.

My subjective opinion: having an entire section dedicated to _not_ over engineer the solutions gives a bit too much away.